### ISSUE TITLE
NGINX logs critical for inactive but purged cache entries

### ISSUE THREAD

**Issue by witrin** [nginx-modules/ngx_cache_purge#19]
I know this is an common problem with NGINX (https://www.miklix.com/nginx/deleting-nginx-cache-puts-critical-unlink-errors-in-error-log/) but is it possible to sync with NGINX internals after `PURGE` so this "critical" error won't be logged?

**Comment by denji** [comment]
Yes, we need to study this question about how https://github.com/nginx-modules/nginx-selective-cache-purge-module/blob/805ca25/src/ngx_selective_cache_purge_module.c#L348-L386

**Comment by painslie** [comment]
@denji Are you looking at completing this enhancement some time this year?  Would be really nice to sync NGINX's internal keystone with the PURGE.
